eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -86,7 +86,11 @@ steps:
 
               # Set non-cone mode otherwise path filters will not work in git >= 2.37.0
               # See https://github.blog/2022-06-27-highlights-from-git-2-37/#tidbits
-              git sparse-checkout set --no-cone '/*' '!/*/' '/eng'
+              # '/*' '!/*/' -> only checkout files in top level directory
+              # '/eng' -> checkout required eng/ scripts/configs
+              # '.config/1espt' -> required for files like .config/1espt/PipelineAutobaseliningConfig.yml used by 1es PT scripts
+              # '.config/guardian' -> required for files like .config/guardian/.gdnbaselines used by 1es PT scripts
+              git sparse-checkout set --no-cone '/*' '!/*/' '/eng' '/.config/1espt' '/.config/guardian'
```

Comment:
Should we just always include /.config directory?

Reply:
Yeah, probably. I'll switch it